packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -88,34 +90,26 @@ export class OrgList implements vscode.Disposable {
         // scratch orgs parented by other (non-default) devHub orgs
         continue;
       }
-      const isExpired =
-        authFields && authFields.expirationDate
-          ? today >= new Date(authFields.expirationDate)
-          : false;
-
-      const aliases = await ConfigUtil.getAllAliasesFor(orgAuth.username);
-      let authListItem =
-        aliases && aliases.length > 0
-          ? `${aliases.join(',')} - ${orgAuth.username}`
-          : orgAuth.username;
 
-      if (isExpired) {
-        authListItem += ` - ${nls.localize(
-          'org_expired'
-        )} ${String.fromCodePoint(0x274c)}`; // cross-mark
+      if (orgAuth.isExpired === true) {
+        // If the scratch org is expired we don't want to see it in the org picker
+        continue;
       }
+      const aliases = orgAuth.aliases || [];
+      const authListItem =
+        aliases?.length > 0
```

Comment:
Don't need the '?' now that aliases is always an array
```suggestion
        aliases.length > 0
```